def test_kasitli_hata():
    assert False, 'Kasıtlı hata mesajı burada'